File: TopCoder03/Ruby/Main.rb
class Cryptography
  def encrypt(numbers)
    let = 1
    numbers.sort!
    numbers[0] += 1
    for number in numbers
      let *= number
    end
    return let
  end
end

numbers = [1000,999,998,997,996,995]
puts Cryptography.new.encrypt(numbers)

File: TopCoder03/PHP/Main.php
<?php
class Cryptography {
	public function encrypt($numbers) {
		$let = 1;
		sort($numbers);
		$numbers[0]++;
		foreach ($numbers as $num) {
			$let *= $num;
		}
		return $let;
	}

}

$numbers = array(1000, 999, 998, 997, 996, 995);
$c = new Cryptography;
printf("%.0f", $c -> encrypt($numbers));
?>